ill-fated-ducks/workforce-management
# Rock seed data2

## Status
Choose:
**READY**


## Description
Re-did my Seed Data file to 

## Todos

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. Verify Data already in DB
2. dotnet run
3. verify that repeat data doesn't exist

## Impacted Areas in Application
List general components of the application that this PR will affect:

## Link to Feature Ticket
#
@ill-fated-ducks
